src/components/Switch.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useMount } from '../hooks/useMount';
+import { toggleTheme } from '../actions';
+
+export const SwitchButton = () => {
+  const [value, setvalue] = useState(false);
```

Comment:
chore: minor typo on camelCase convention
```suggestion
  const [value, setValue] = useState(false);
```

Reply:
Fixed.